backend/surety/surety-persister.yml

```
@@ -0,0 +1,97 @@
+        - query: INSERT INTO dristi_surety_document(id,fileStore,documentUid,documentType,isActive,surety_id, additionalDetails) VALUES (?, ?, ?, ?, ?, ?, ?) ON CONFLICT (id) DO UPDATE SET fileStore = EXCLUDED.fileStore, documentUid = EXCLUDED.documentUid, documentType = EXCLUDED.documentType, isActive=EXCLUDED.isActive, surety_id = EXCLUDED.surety_id, additionalDetails = EXCLUDED.additionalDetails;
+          basePath: surety.documents.*
+          jsonMaps:
+            - jsonPath: $.surety.documents.*.id
+            - jsonPath: $.surety.documents.*.fileStore
+            - jsonPath: $.surety.documents.*.documentUid
+            - jsonPath: $.surety.documents.*.documentType
+            - jsonPath: $.surety.documents.*.isActive
+            - jsonPath: $.surety.id
+            - jsonPath: $.surety.documents.*.additionalDetails
+              type: JSON
+              dbType: JSONB
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)